Neos.Fusion/Tests/Functional/FusionObjects/DataStructureTest.php

```diff
@@ -139,10 +139,40 @@ public function untypedChildKeysWorkWithFusionPositionSorting(): void
     /**
      * @test
      */
-    public function unsetUntypedChildKeysWillRenderAsDataStructure(): void
+    public function unsetChildKeyWillNotRender(): void
     {
         $view = $this->buildView();
-        $view->setFusionPath('dataStructure/unsetUntypedChildKeyWillRenderAsDataStructure');
-        self::assertEquals(['buz' => 456, 'keyWithUnsetType' => ['bat' => 123]], $view->render());
+        $view->setFusionPath('dataStructure/unsetChildKeyWillNotRender');
+        self::assertEquals(['foo' => 'bar'], $view->render());
+    }
+
+    /**
+     * @test
+     */
+    public function unsetUntypedChildKeyWillNotRender(): void
+    {
+        $view = $this->buildView();
+        $view->setFusionPath('dataStructure/unsetUntypedChildKeyWillNotRender');
+        self::assertEquals(['buz' => 456], $view->render());
+    }
+
+    /**
+     * @test
+     */
+    public function nulledChildKeyWillRenderAsNull(): void
+    {
+        $view = $this->buildView();
+        $view->setFusionPath('dataStructure/nulledChildKeyWillRenderAsNull');
+        self::assertEquals(['foo' => 'bar', 'null2' => null], $view->render());
+    }
```

Comment:
This test mainly documents the current behavior:

The fusion code is:
```fusion
dataStructure.nulledChildKeyWillRenderAsNull = Neos.Fusion:DataStructure {
  foo = 'bar'
  null1 = null
  null2 = ${null}
}
```

And `null1` is removed by the `PositionalArraySorter` – This is a separate issue that could be resolved with #3350